apps/web/server/routers/viewer.tsx

```diff
@@ -891,6 +893,32 @@ const loggedInViewerRouter = createProtectedRouter()
         throw new TRPCError({ code: "BAD_REQUEST" });
       }
     },
+  })
+  .mutation("submitFeedback", {
+    input: z.object({
+      rating: z.string(),
+      comment: z.string(),
+    }),
+    async resolve({ input, ctx }) {
+      const { rating, comment } = input;
+
+      const feedback = {
+        userId: ctx.user.id,
+        rating: rating,
+        comment: comment,
+      };
+
+      await ctx.prisma.feedback.create({
+        data: {
+          date: dayjs().toISOString(),
+          userId: ctx.user.id,
+          rating: rating,
+          comment: comment,
+        },
+      });
+
+      if (process.env.SEND_FEEDBACK_EMAIL && comment) sendFeedbackEmail(feedback);
+    },
```

Comment:
@zomars @emrysal let me know if this is the right place or if there could be any improvements.